scripts/otel_packaging.py

```diff
@@ -28,12 +29,18 @@ def get_instrumentation_packages():
         if not os.path.isdir(pkg_path):
             continue
 
-        version = subprocess.check_output(
-            "hatch version",
-            shell=True,
-            cwd=pkg_path,
-            universal_newlines=True,
-        )
+        try:
+            version = subprocess.check_output(
+                "hatch version",
+                shell=True,
+                cwd=pkg_path,
+                universal_newlines=True,
+            )
+        except CalledProcessError as e:
```

Comment:
Please change the exception name so the pylint will pass:
C0103: Variable name "e" doesn't conform to '(([a-z_][a-z0-9_]{1,})|(_[a-z0-9_]*)|(__[a-z][a-z0-9_]+__))$' pattern ('(([a-z_][a-z0-9_]{1,})|(_[a-z0-9_]*)|(__[a-z][a-z0-9_]+__))$' pattern) (invalid-name)